index.test-d.ts

```diff
@@ -167,11 +167,11 @@ expectType<{
 
 const baseLevelFour = new BaseLevelFour();
 
+// See the node on static defaults in index.d.ts for why defaultFour is missing
+// .options from .defaults() is only supported until a depth of 4
 expectType<{
   version: string;
   defaultOne: string;
   defaultTwo: number;
   defaultThree: string[];
-  defaultFour: number;
-  // @ts-expect-error - .options from .defaults() is only supported until a depth of 4
```

Comment:
why did you remove it? Maybe we find a fix for this in future at which point this line will fail with `tsd` and remind us to update tests & docs?

Reply:
Ah, sorry, I'd meant to leave a note - `@ts-expect-error` could apply to the whole assertion. I'll duplicate to make sure we get both checks.